### PR TITLE
Add the ability to add additional patches before applying trypatch in…

### PR DESCRIPTION
**Pull request by vaibhavmagarwal**
… bisection_script.

### CODE REVIEW

**Comment by vaibhavmagarwal** [comment]
With this patch, one will be able to apply topmost patch in the series with -p option before applying trypatch and then it will be pushed to try.
r? @jmaher

File: bisection_script.py [hunk 1]
```diff
@@ -25,9 +25,13 @@ def parse_args(argv=None):
...
                         type=str,
                         help="Current changeset.")
+    parser.add_argument("-p", "--patch",
+                        dest="patch",
+                        type=str,
+                        help="Topmost additional patch in series to push before trypatch.")
     parser.add_argument("--cwd",
                         dest="cwd",
-                        #required=True,
+                        required=True,
```
Comment (jmaher) [inline bisection_script.py]: why is this required now?

Comment (vaibhavmagarwal) [inline bisection_script.py]: Actually required is necessary, since the commands below will fail to apply if cwd of inbound is not given. I meant to put it in previous patch, just forgot to uncomment it.
